client/state/partner-portal/partner/selectors.ts

```diff
@@ -61,7 +61,8 @@ export function isAgencyUser( state: PartnerPortalStore | IAppState ): boolean {
 	const currentUser = getCurrentUser( state );
 	return (
 		( currentUser?.jetpack_partner_types?.includes( 'agency' ) ||
-			currentUser?.jetpack_partner_types?.includes( 'agency_beta' ) ) ??
+			currentUser?.jetpack_partner_types?.includes( 'agency_beta' ) ||
+			currentUser?.jetpack_partner_types?.includes( 'a4a_agency' ) ) ??
```

Comment:
Can we create a new selector to check if `jetpack_partner_types` includes `a4a_agency` only. As per my conversion here: p1718095623367579/1717772046.767839-slack-C06JY8QL0TU, it looks like it will always have `a4a_agency`. 

Reason being: the `isAgencyUser` selector is being used throughout Jetpack Manage.